pkg/storaged/vgroup-details.jsx

```diff
@@ -109,7 +109,7 @@ class VGroupSidebar extends React.Component {
             return (
                 <tr key={pvol.path}>
                     <td className="storage-icon">
-                        img src="images/storage-disk.png" /
```

Comment:
I guess it would present it as a string, that's why you didn't receive warning :)